content/docs/command-reference/remote/default.md

```diff
@@ -22,6 +22,16 @@ You can query/set/replace/unset the _default remote_ using the options of this
 command. If the `name` of the remote is not provided and `--unset` is not
 specified, this command returns the name of the default remote.
 
+When reading (`name` is not provided), the value ise read from the system,
+global, project and local configuration files by default, and options
+`--system`, `--global`, `--project`, and `--local` can be used to tell the
+command to read from only that location.
+
+When writing (`name` is provided), the new value is written to the projects
+configuration file by default (`.dvc/config`), and options `--system`,
+`--global` and `--local` can be used to tell the command to write to that
+location (you can say `--project` but that is the default).
```

Comment:
Let's generalize copying (ad hoc) from the `config` ref?

Reply:
Also maybe this text is more important to have in `remote modify`. Not sure about here (may be TMI and it can be found in the **Options**) but up to you. It would need to be in `remote rename` too though.